webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/Table/Table.js

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  TableComposable,
+  Thead,
+  Tr,
+  Th,
+  Tbody,
+  Td,
+  ActionsColumn,
+} from '@patternfly/react-table';
+import { translate as __ } from '../../../../common/I18n';
+import { useTableSort } from '../../Helpers/useTableSort';
+import Pagination from '../../../Pagination';
+import { DeleteModal } from './DeleteModal';
+import EmptyPage from '../../../../routes/common/EmptyPage';
+
+export const Table = ({
+  columns,
+  errorMessage,
+  getActions,
```

Comment:
I don't see this to come from TableIndexPage, nor being documented, how can it be used/what for?